Apps/Database/src/Delegates/DBGenericCacheDelegate.cs

```diff
@@ -157,6 +157,7 @@ public DBResult<GenericCache> AddCacheObject(GenericCache cacheObject, bool comm
                 }
                 catch (DbUpdateException e)
                 {
+                    this.dbContext.Entry(cacheObject).State = EntityState.Detached;
```

Comment:
nice finding. It is a bit annoying with EF-Framework due to these contextual links